egs/wsj/s5/steps/data/reverberate_data_dir.py

```diff
@@ -84,9 +84,6 @@ def GetArgs():
     return args
 
 def CheckArgs(args):
-    if not os.path.exists(args.output_dir):
```

Comment:
@vimalmanohar I don't see why. This part of the code is only creating output dir if it does not  exist. Anyways, I am modifying the code to fail if the output directory already exists.

Reply:
I don't think Vimal was asking for it to fail if the output dir exists (and I don't recommend it either).
I think he's pointing out that since, below, you do os.makedirs with no such guard, it will fail if it did not
exist.  It seems to have an `exist_ok` parameter which you can set to true.

Reply:
The original version of the script will re-create the augmented data dir if the script is run again. But Phani modified it so it would do nothing if run again, which I think is unexpected behavior. It should either 1) re-create the augmented data dir, 2) fail with error saying it already exists.